db/migrate/20170421153056_add_email_count_to_signatures.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailCountToSignatures < ActiveRecord::Migration
+  def change
+    add_column :signatures, :email_count, :integer
```

Comment:
If we add a default 0 here we can drop the `def email_count; super || 0; end` in `Signature`